.github/workflows/git-dev.yml

```
@@ -0,0 +1,80 @@
+name: Publish Git-Dev
+
+on:
+  push:
+    branches:
+        - v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)